go/ql/lib/semmle/go/dataflow/FunctionInputsAndOutputs.qll

```diff
@@ -74,7 +74,17 @@ private class ParameterInput extends FunctionInput, TInParameter {
 
   override predicate isParameter(int i) { i = index }
 
-  override DataFlow::Node getEntryNode(DataFlow::CallNode c) { result = c.getArgument(index) }
+  override DataFlow::Node getEntryNode(DataFlow::CallNode c) {
+    result = getArgument(c, index) and
```

Comment:
Intentionally adding method receivers through use of `getArgument` not `c.getArgument`?

Reply:
No, that was a copy and paste error 😬